test/sampledata/connection_test.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+import pytest
+import requests
+
+from transcriptic.sampledata.connection import MockConnection
+from transcriptic.sampledata.project import sample_project_attr
+from transcriptic.util import load_sampledata_json
+
+
+class TestMockConnection:
+    def test_defaults(self):
+        assert MockConnection().organization_id == "sample-org"
+
+    def test_not_registered(self):
+        with pytest.raises(
+            requests.exceptions.ConnectionError, match="Mocked route not implemented"
+        ):
+            MockConnection().project(project_id="invalid-project")
+        with pytest.raises(
+            requests.exceptions.ConnectionError,
+            match="Connection refused by Responses - the call doesn't "
+            "match any registered mock.",
+        ):
+            MockConnection(verbose=True).project(project_id="invalid-project")
+
+    def test_registered_responses(self):
+        mock_connection = MockConnection()
+        assert mock_connection.project(project_id="p123") == sample_project_attr
+        assert (
+            mock_connection._get_object(obj_id="p123", obj_type="project")
+            == sample_project_attr
+        )
+        assert (
+            mock_connection.projects()
+            == load_sampledata_json("sample-org-projects.json")["projects"]
+        )
+        assert mock_connection.runs(project_id="p123") == load_sampledata_json(
+            "p123-runs.json"
+        )
+
+    def test_jupyter_project(self):
```

Comment:
example usage: If this looks good, will roll this out for the other objects such as Run, Container, etc.